src/antidote_crdt_counter_b.erl

```diff
@@ -220,6 +237,8 @@ equal(BCounter1, BCounter2) ->
 
 -ifdef(TEST).
```

Comment:
Please keep all includes at the top of the file. This is consistently done in all other files.

Reply:
Done